### ISSUE TITLE
Will not run since update of pre-built windows exe pushed today

### ISSUE THREAD

**Issue by SuperSylph** [shadowmoose/RedditDownloader#158]
### Describe the bug
An update of the prebuilt windows executable was pushed today (old exe renamed and replaced), even though no new release is listed on Github. It fails to run with the error given below.

### Environment Info
 - Win 10.0.18363 Build 18363
 - RMD 3.1.0 ? no new version listed, can't run to check if the one pushed is newer

### Screenshots/Information
```
C:\Windows\System32>C:\Software\RMD\RMD-windows.exe
Traceback (most recent call last):
  File "site-packages\PyInstaller\loader\rthooks\pyi_rth_pkgres.py", line 13, in <module>
  File "c:\hostedtoolcache\windows\python\3.7.8\x64\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 623, in exec_module
  File "site-packages\pkg_resources\__init__.py", line 86, in <module>
ModuleNotFoundError: No module named 'pkg_resources.py2_warn'
[27400] Failed to execute script pyi_rth_pkgres
```
### Additional context
None.

**Comment by shadowmoose** [comment]
Ah, lovely. The build tool for RMD is broken with the latest package manager. Great. Well, I guess I'll disable the binary builds until I can unravel this. Thanks for the report.

**Comment by shadowmoose** [comment]
This has been resolved as of 6ac9129 (Release 3.1.1). The build process for binaries has been fixed & upgraded to fully unit-test the compiled binaries before deployment, so that a malformed release cannot happen again.
